packages/guides/src/Renderer/RenderIntersphinxSet.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace phpDocumentor\Guides\Renderer;
+
+use phpDocumentor\Guides\Handlers\RenderCommand;
+use phpDocumentor\Guides\Handlers\RenderDocumentCommand;
+use phpDocumentor\Guides\Handlers\RenderDocumentHandler;
+use phpDocumentor\Guides\RenderContext;
+use phpDocumentor\Guides\Setup\QuickStart;
+use phpDocumentor\Guides\UrlGenerator;
+
+class RenderIntersphinxSet implements RenderSet
+{
+
+    public function supports(string $outputFormat): bool
+    {
+        return $outputFormat === RenderSet::INTERSPHINX;
+    }
+
+    public function render(RenderCommand $renderCommand): void
+    {
+        $inventory = [
+            'std:doc' => []
+        ];
+        $renderContext = RenderContext::forSitemap(
```

Comment:
I think it might be good to spit the RenderContext into 2 separated classes. the only reason you need this here is because of the `relativeDocUrl` method. I'm even thinking if we could extract this function to make it part of the UrlGenerator.

Splitting will give you the benefit of the non-nullable `DocumentNode` in the context.